docs/react/introduce.en-US.md

```diff
@@ -0,0 +1,95 @@
+---
+order: 0
+title: Ant Design of React
+---
+
+antd is a set of React components which follow Ant Design specification. It is designed to develop RIA sucn as dashboard to serve enterprise.
+
+<div class="pic-plus">
+  <img width="150" src="https://t.alipayobjects.com/images/rmsweb/T11aVgXc4eXXXXXXXX.svg">
+  <span>+</span>
+  <img width="160" src="https://t.alipayobjects.com/images/rmsweb/T16xRhXkxbXXXXXXXX.svg">
+</div>
+
+<style>
+.pic-plus > * {
+  display: inline-block!important;
+  vertical-align: middle;
+}
+.pic-plus span {
+  font-size: 30px;
+  color: #aaa;
+  margin: 0 20px;
+}
+</style>
+
+---
+
+## Features
+
+- Designed as Ant Design, a design language to create user friendly and beautiful websites.
+- It is a set of high quality UI components and based on [React Component](http://react-component.github.io/badgeboard/).
+- Provide a work flow which is based on npm and webpack and babel, it supports ES2015 and TypeScript.
+
+## Installation
+
+```bash
+$ npm install antd
+```
+
+## Example
+
+```jsx
+import { DatePicker } from 'antd';
+ReactDOM.render(<DatePicker />, mountNode);
+```
+
+Import style:
+
+```jsx
+import 'antd/dist/antd.css';  // or 'antd/dist/antd.less'
+```
+
+You can use `import DatePicker from 'antd/lib/date-picker'` or [babel-plugin-antd](https://github.com/ant-design/babel-plugin-antd)(This plugin supports import component and styles on demand) to import components on demand.
```

Comment:
You can use:
- `import DatePicker from 'antd/lib/date-picker';` 
- `import { DatePicker } from 'antd';` when [babel-plugin-antd](https://github.com/ant-design/babel-plugin-antd) is also used.

This plugin supports importing components and styles on demand.